packages/credential-showcase-openapi/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "postinstall": "mvn clean generate-sources",
+    "postinstall": "pnpm generate-models",
```

Comment:
we generate with MVN, and we will publish it, but for me is fine to keep a local version